apps/betterangels-backend/accounts/managers.py

```diff
@@ -9,25 +9,24 @@
 
 
 class UserManager(BaseUserManager["User"]):
-    def create_user(self, email: str, password: str = "", **extra_fields: Any) -> "User":
-        if not email:
-            raise ValueError("The Email field must be set")
+    def create_user(self, **extra_fields: Any) -> "User":
         extra_fields.setdefault("is_staff", False)
         extra_fields.setdefault("is_superuser", False)
-        email = self.normalize_email(email)
+        email = self.normalize_email(extra_fields.pop("email", None))
         user = self.model(email=email, **extra_fields)
-        user.set_password(password)
+        user.set_password(extra_fields.get("password"))
         user.save(using=self._db)
+
         return user
 
-    def create_client(self, email: str, **extra_fields: Any) -> "User":
+    def create_client(self, **extra_fields: Any) -> "User":
         random_id = uuid.uuid4()
-        email = email or self.normalize_email(f"client_{random_id}@example.com")
-        client = self.create_user(email, username=random_id, **extra_fields)
+        client = self.create_user(username=random_id, **extra_fields)
         client.set_unusable_password()
+
         return client
 
-    def create_superuser(self, email: str, password: str = "", **extra_fields: Any) -> "User":
```

Comment:
Are we not going to conform the the base class's interface anymore? 

https://github.com/django/django/blob/89557d4c66b469616fc3a16ba11b2999233efa8d/django/contrib/auth/models.py#L164